CHANGES.rst

```diff
@@ -5,7 +5,9 @@
 4.3.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- On removal of a site remove the sub of its site manager on the parent site
```

Comment:
This feels like it should at least be a 4.4.0 to me?

Reply:
Done.